ce_build.sh

```diff
@@ -34,6 +36,7 @@ docker run -v "${PWD}/shared:/shared" \
   -e PACKAGE_LANG="${PACKAGE_LANG:-cpp}" \
   -e GAZEBO_VERSION="${GAZEBO_VERSION:-7}" \
   -e DOCKER_BUILD_SCRIPT="${DOCKER_BUILD_SCRIPT}" \
+  -e WORKSPACES="${WORKSPACES}" \
```

Comment:
Can be add a sensible default to this? That could be performed here (e.g with `:-`) or in ros1_sa_build.sh. Does it work with robot_ws and simulation_ws as the defaults?